src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,9 @@
-const booleanToWord = boolean => {};
-
+const booleanToWord = (boolean) => {
+  if (boolean === true) {
+    return "Yes";
```

Comment:
The task is to return yes or no. But understand that returning something with more understandable for the reader to see. 
What I am not unsure on what to do if it is a number or another string?

Reply:
It may be out of scope for this task, just its always worth considering things like this though! The most common way to handle the input not being of the type expected would be to check it at the start of the function and throw an error if its not whats expected before doing and performing any logic on it:
```
if (typeof boolean !== "boolean") {
  throw new Error("Invalid type supplied, boolean required");
}
```